doc/SUMMARY.md

```diff
@@ -0,0 +1,99 @@
+# Summary
+
+- [architecture](./architecture/README.md)
+  - [context](./architecture/context.md)
+  - [database_schemata](./architecture/database_schemata.md)
+  - [glossary](./architecture/glossary.md)
+  - [sequence](./architecture/sequence.md)
+  - [solution](./architecture/solution.md)
+- [authors](./authors.md)
+- [ci](./ci.md)
+- [contrib](./contrib/README.md)
+  - [backend](./contrib/backend.md)
+  - [frontend](./contrib/frontend.md)
+- [decisions](./decisions/README.md)
```

Comment:
Do we really have to list each one of them? Can we somehow include the whole folder?